src/Markdown.MAML/Markdown.MAML.csproj

```diff
@@ -1,123 +1,10 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{9AFC25E8-6BAF-4B6B-B33F-CC2E0496E9C0}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Markdown.MAML</RootNamespace>
-    <AssemblyName>Markdown.MAML</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFramework>netstandard1.6</TargetFramework>
```

Comment:
Shouldn't we be targeting both net452 and netstandard1.6? We still need this to work on PowerShell 5.1 right?

Reply:
It seems to work as is: AppVeyor CI runs full PowerShell 5.1

But I would not claim that I understand dotnet taxonomy. If it breaks on older versions of PS, we can revisit this.

Reply:
Fairly sure this is because the framework is installed. I'll confirm on a fresh machine.

Reply:
Looks like you are right, got that on a fresh server 2016 machine. Good catch!

```
Import-Module : Could not load file or assembly 'System.Runtime, Version=4.1.0.0, Culture=neutral,
PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
At line:1 char:1
+ Import-Module C:\Users\localadmin\Downloads\platyPS-0.7.6.1064\Markdo ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Import-Module], FileNotFoundException
    + FullyQualifiedErrorId : System.IO.FileNotFoundException,Microsoft.PowerShell.Commands.ImportModuleCommand
```

Reply:
So is there a way to build 1 assembly that will work on ps 5.1 and 6.0?
Or [two assemblies hack](https://github.com/OneGet/oneget/blob/f975edb635ddb4265862be71c98c8b85a11345bb/src/Microsoft.PowerShell.PackageManagement/PackageManagement.psm1#L10) is the only way?

cc @daxian-dbw

Reply:
I guess one way would be to build only net451 version and ship it (because it works just fine on pwsh for some reason), and for local development on non-windows platform build netstandart1.6
but that doesn't feel too good

Reply:
Two assemblies for sure, can't see that as a hack though. It is how every other cross plat module is built afaik

Reply:
well, you can use the currently published modules from the gallery (built for 4.5) **just fine** on pwsh. At least I haven't seen problems so far and problems are usually showing up very early when it cannot resolve `System.Runtime` and friends. Also a bunch of dlls, like makdig are working ok on both, when consumer thru `import-module` (it's also built for full). So we can still build for full and publish it as the one package.